.circleci/config.yml

```diff
@@ -40,7 +45,30 @@ jobs:
       - store_artifacts:
           path: coverage/lcov-report/
 
+      - run: npm run test:performance:ci
+
+      - store_artifacts:
+          path: performance-tests/__snapshots__
+
       - run: npm run build
 
       - store_artifacts:
           path: dist
+
+      - run:
+          name: Commit Assets
+          command: |
+            # configuring user
+            git config user.email "j.r.jurman@gmail.com"
+            git config user.name "CircleCI Job"
+
+            if [[ -z "${CIRCLE_PULL_REQUEST}" ]]
+            then
+              echo "Master Build, checkout to master"
+              git checkout
+            else
+              echo "PR Build, checkout to new branch"
+              git checkout -b ${CIRCLE_BRANCH}-artifacts
+            fi
+            git commit --allow-empty -am "Update Snapshots & Badges [skip ci]"
```

Comment:
`[skip ci]` is important here, this actually prevents CircleCI from infinitely triggering builds.